src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj

```diff
@@ -27,4 +27,16 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
+    <None Update="TestData\IntegrationAccountAssemblyContent.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="TestData\IntegrationAccountCertificate.cer">
```

Comment:
Is this an authentication certificate? Not sure if we can check this in 
@shahabhijeet  FYI

Reply:
This was a file that had already existed in the repo to be used in tests. I'm not sure how the tests worked without the file being included. They don't work locally when I don't include it. In general this isn't a cert to anything in particular, just a cert in general.

Reply:
@refortie if this is a self-signed cert, this should be fin.